packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -213,6 +221,14 @@ contract ValidatorManager is ISemver, IValidatorManager {
         _activateValidator(msg.sender);
     }
 
+    /**
+     * @inheritdoc IValidatorManager
+     */
+    function tryActivateValidator(address validator) external onlyAssetManager {
+        if (getStatus(validator) == ValidatorStatus.READY && !inJail(validator))
```

Comment:
Why don't we make this as a modifier? Then, we can reuse this to `activateValidator()` func.

Reply:
Since this function does not revert even if the condition is not met, I think it's not appropriate to use modifier here.